src/Notifications/NotificationsViewExtension.cs

```diff
@@ -51,15 +56,19 @@ public void Loaded(ViewLoadedParams p)
         {
             viewLoadedParams = p;
             dynamoWindow = p.DynamoWindow;
+            var viewModel = dynamoWindow.DataContext as DynamoViewModel;
```

Comment:
ugh, wish we could avoid this. What about adding the logger to the set of objects we pass into the extensions and make available?

Reply:
We pass stuff into view extensions via `ViewLoadedParams` or `ViewStartParams` and their base classes and I thought we agreed it isn't a good idea to add any new property to these classes so i don't see how I can make the logger available to this extension, you?

Reply:
I feel we need to agree on a strategy since people are going to read our code and all implement similar way than us. This part is impacting my PR as well

Reply:
What I said is that we should not add things to the API which are not useful to all extensions. Logger seems more generally useful - but like you said, notifications can already log messages - though it's very hard to discover.

Reply:
I think the design originally conceived for enabling logging from extensions was to inherit from/implement `ILogSource` but unfortunately `NotificationViewExtension` does not implement `ILogSource`. So for those extension that already do, giving access to the logger may be redundant.